src/content/docs/apis/nerdgraph/examples/apm-config-nerdgraph.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "NerdGraph tutorial: APM agent configuration examples"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Examples of using New Relic NerdGraph API to adjust APM 
+redirects:
+  - /docs/apis/nerdgraph/examples/export-dashboards-pdfpng-api-tutorial
+---
+
+This doc is a place for examples of configuring APM agents using our NerdGraph API. 
+
+## Configure server-side configuration [#server-side-config]
+
+Note that for APM agents to use configuration values changed via NerdGraph, [server side configuration](/docs/apm/agents/manage-apm-agents/configuration/server-side-agent-configuration#requirements) must be enabled.
+
+For requirements, see [server-side config requirements](/docs/apm/agents/manage-apm-agents/configuration/server-side-agent-configuration#requirements).
+
+Here's an example query returning the status of the server side configuration setting for a given entity. 
+
+```
+query ExampleReadQuery {
+  actor {
+    entity(guid:"ZjY1ODgxfEFQTXxBUFBYSUNBVElPTnz0ODEwMTY3NzZ") {
+      ...on ApmApplicationEntity {
+        apmSettings {
+          apmConfig {
+            useServerSideConfig
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Here's an example of disabling [server-side configuration](/docs/apm/agents/manage-apm-agents/configuration/server-side-agent-configuration): 
+
+```
+mutation ExampleUpdateQuery($settings: [AgentApplicationSettingsUpdateInput!]!) {
+  agentApplicationSettingsUpdate(settings: $settings) {
+    apmSettings {
+      apmConfig {
+        useServerSideConfig
+      }
+    }
+    errors {
+      description
+    }
+  }
+}
+```
+
+And here's an example of what might be used for the `settings` query variables. Note that this uses an array, which may be helpful if you have a lot of entities you want to update. 
```

Comment:
These variables are fundamentally linked to the above code. The above code doesn't turn off the setting per se, the values in the settings JSON below is what determines that.

So, maybe (above): "Here is an example to update the value of useServerSideConfig" and (below) "Here is the JSON for the `$settings` variable to turn server side config off using the above query"

Reply:
i attempted to combine these.